__tests__/endpoint.test.js

```diff
@@ -3,180 +3,211 @@ const data = require("../db/data/test-data");
+      }
     });
-    test('GET:404 sends an appropriate status and error message when given a valid but non-existent id', () => {
+
     return request(app)
-        .get('/api/articles/999/comments')
-        .expect(404)
-        .then(({body}) => {
-        expect(body.msg).toBe('comments not found');
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        articles.forEach((article) => {
+          if (article.comment_count != "0") {
+            expect(article.comment_count).toBe(
+              String(commentCountObject[article.article_id])
+            );
+          }
         });
-    });
-    test('GET:400 sends an appropriate status and error message when given an invalid id data-type', () => {
+      });
+  });
+});
+describe("/api/articles/:article_id/comments", () => {
+  test("GET:200 sends an array of comments to the client with an article_id of 1", () => {
     return request(app)
-        .get('/api/articles/not-a-number/comments')
-        .expect(400)
-        .then(({body}) => {
-        expect(body.msg).toBe('bad request');
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments.length).toBe(11);
+        body.comments.forEach((comment) => {
+          expect(comment.article_id).toBe(1);
+          expect(typeof comment.comment_id).toBe("number");
+          expect(typeof comment.votes).toBe("number");
+          expect(typeof comment.author).toBe("string");
+          expect(typeof comment.body).toBe("string");
+          expect(typeof comment.created_at).toBe("string");
         });
-    });
-    test('GET:200 array is sorted with most recent comments first', () => {
-        return request(app)
-          .get("/api/articles/1/comments")
-          .expect(200)
-          .then(({body}) => {
-            expect(body.comments).toBeSortedBy('created_at')
-          })
-    })
-})
+      });
+  });
+  test("GET:200 responds with an empty array when a valid, existing article_id is specified but there are no comments attached to it", () => {
+    return request(app)
+      .get("/api/articles/4/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments).toEqual([]);
+      });
+  });
+  test("GET:404 sends an appropriate status and error message when given a valid but non-existent id", () => {
+    return request(app)
+      .get("/api/articles/999/comments")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("comments not found");
+      });
+  });
+  test("GET:400 sends an appropriate status and error message when given an invalid id data-type", () => {
+    return request(app)
+      .get("/api/articles/not-a-number/comments")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("bad request");
+      });
+  });
+  test("GET:200 array is sorted with most recent comments first", () => {
```

Comment:
I would probably have a length assertion in this test too just to show there are multiple things in the array. This will give more confidence to your sortedBy.